benchmarks/benchmarks/bench_ufunc.py

```diff
@@ -60,6 +60,31 @@ def time_ufunc_types(self, ufuncname):
         [self.f(*arg) for arg in self.args]
 
 
+class UFuncBenchmark(Benchmark):
+    params = ['bool', 'uint8', 'int8', 'int32', 'uint64', 'int64', 'float16',
+              'float32', 'float64', 'complex128', 'datetime64[ns]',
+              'timedelta64[ns]']
+    param_names = ['dtype']
+
+    def setup(self, dtype):
+        self.arr = np.full(100000, 1, dtype=dtype)
+        self.val = self.arr[0]
+
+
+class MinMax(UFuncBenchmark):
+    def time_fmax(self, dtype):
+        np.fmax(self.arr, self.arr)
+
+    def time_fmin(self, dtype):
+        np.fmin(self.arr, self.arr)
```

Comment:
```suggestion
        np.fmin(self.arr, self.arr_2, out=self.out)
```

using `arr` as both inputs is something of an edge case. Using the out parameter will ensure you aren't allocating a new array (which can be slow or fast depending on the previous state of the RAM).